README.md

```diff
@@ -54,22 +106,42 @@ conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch-nightly
 conda install pytorch torchvision torchaudio cpuonly -c pytorch-nightly
 ```
 
-#### Dependencies: PyEnv
-
+For a `pyenv`/`virtualenv` setup, one could do:
 ```shell
-# install libpython.a with -fPIC enabled
 export CFLAGS="-fPIC -g"
-pyenv install --force 3.8.6
+~/.pyenv/bin/pyenv install --force 3.8.6
 virtualenv -p ~/.pyenv/versions/3.8.6/bin/python3 ~/venvs/multipy
 source ~/venvs/multipy/bin/activate
+pip install -r dev-requirements.txt
 
 # cuda
 pip3 install --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cu113
 
 # cpu only
 pip3 install --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
+
+```
+
+#### Running `pip install`
+
+Once all the dependencies are successfully installed, most importantly including a PIC-library of python and the latest nightly of pytorch, we can run the following, in either `conda` or `virtualenv`, to install both the python modules and the runtime/interpreter libraries:
```

Comment:
should this be moved under building from source? The normal pip install should cover this?

Reply:
This is the last step of the "installing with pip install" section.

Reply:
Thinking of renaming "Installing from source" to something else, as it can be potentially confusing and `pip install` is basically doing the same thing.